### ISSUE TITLE
Suggestion: Add ability for players to toggle the vehicle on and off

### ISSUE THREAD

**Issue by ICronyI** [xaukael/vehicle-tokens#4]
Hello! love your module, but i found a minor issue with it:

It seems like only the GM can toggle the attachment of the tokens to the vehicle. I think it would be great if the players were allowed to attach themselves to it without the aid of the gm !

Thanks in advance for any answers and for your time!

**Comment by xaukael** [comment]
Attaching to the vehicle requires that you have permission to edit the token. They will need ownership of the vehicle. Thoughts on whether they should be able to attach tokens they do not own to theirs?

**Comment by ICronyI** [comment]
@xaukael Yeah i tried it with them owning the vehicle, but they don't have the option to attach as of now i think?

I think it should be fine to allow them to attach tokens they do not own to theirs; worse case scenario you could maybe make it so that if they try to attach another character's token to the vehicle, it create a popup on the other player's game, asking if they are ok with it, and they could detach themselves at any moment if they so wish?

Another option could be to switch it around; the players could choose to attach and detach their own token one by one to the vehicle by themselves. Would require a bit more time to get everyone onboard but it would avoid any possible issue of people attaching other tokens to vehicles unwillingly

**Comment by ICronyI** [comment]
Any news on this?
